skcosmo/feature_selection/simple_cur.py

```diff
@@ -0,0 +1,183 @@
+import numpy as np
+
+from sklearn.utils import check_random_state
+from sklearn.utils.extmath import randomized_svd
+
+from ._greedy import GreedySelector
+from ..utils.orthogonalizers import X_orthogonalizer, Y_feature_orthogonalizer
+
+
+class SimpleCUR(GreedySelector):
```

Comment:
naming: should `SimpleCUR` be named `GreedyCUR`/`GreedyCURSelection` instead? And same for FPS

Reply:
All CUR / FPS are greedy by definition. We're using `Simple` here to delineate `Simple` versus `PCov` selection. I'm open to `UnsupervisedFPS` or something of the like though

Reply:
I thought that SimpleFPS is in contrast with Voronoi_FPS, not the PCov_FPS, but it's a details. Maybe `FPS` and `CUR` is enough? And all other classes are called `Voronoi_FPS`,  `PCov_FPS` and `PCov_CUR`?

Reply:
Why not just `FPS`/`FpsSelection` then? And the Pcov one would be `PCovFPS`

Reply:
Yep, I agree with the last one

Reply:
Okay, I'll update!